src/containers/my-resources/lessons-container/LessonsContainer.tsx

```diff
@@ -81,18 +69,14 @@ const LessonsContainer = () => {
     navigate(createUrlPath(authRoutes.myResources.editLesson.path, id))
   }
 
-  const { response, loading, fetchData } = useAxios<
-    ItemsWithCount<Lesson>,
-    GetResourcesParams
-  >({
-    service: getMyLessons,
-    defaultResponse: defaultResponses.itemsWithCount,
-    onResponseError
-  })
-
   const props = {
     columns: columnsToShow,
-    data: { response, getData: fetchData },
+    data: {
+      response: isSuccess ? response : defaultResponses.itemsWithCount,
+      getData: () => {
```

Comment:
can you just define it like below?
`getData: refetch`

Reply:
fixed